bin/update

```diff
@@ -9,8 +9,10 @@ cd client && yarn
 echo "Updating server dependencies:"
 cd ../server && yarn
 
-echo "Set up env variables:"
-cp ../server/.env.sample ../server/.env
```

Comment:
@AleksSenkou looks like it should be in the updated version of this file

Reply:
@datarockets/cuties ideally we need to copy the whole file only for development start, after that, we need only to add new env variables from `.env.sample` to `.env`

Here is a prototype for ruby:
bin/setup: https://github.com/datarockets/bevital/blob/master/bin/setup#L23
bin/update: https://github.com/datarockets/bevital/blob/master/bin/update#L37

Thoughts about having something like that in the future?

Reply:
@AleksSenkou I agree with that approach